src/main/resources/com/linecorp/armeria/server/docs/assets/js/armeria.js

```diff
@@ -24,6 +24,23 @@ $(function () {
     render(URI(window.location.href));
   });
 
+  function updateDocString(node) {
+    if (node instanceof Array) {
+      for (var idx in node) {
+        updateDocString(node[idx]);
+      }
+    } else if (node instanceof Object) {
+      var docString = node['docString'];
+      if (docString && typeof docString === 'string') {
+        node['docString'] = docString.replace(/\n/gi, '<br>');
```

Comment:
and yeah, of course, we should do some security stuff like allowing a certain set of tags only.